wgpu-hal/src/dx12/command.rs

```diff
@@ -418,18 +433,21 @@ impl crate::CommandEncoder<super::Api> for super::CommandEncoder {
         };
         let raw_format = conv::map_texture_format(dst.format);
 
+        let block_size = dst.format.describe().block_dimensions.0 as u32;
         for r in regions {
-            let src_box = make_box(&wgt::Origin3d::ZERO, &r.size);
+            let uprounded_size = upround_extent(r.size, block_size);
```

Comment:
Something is not right here. WebGPU spec [requires](https://gpuweb.github.io/gpuweb/#valid-texture-copy-range) that copy size is already a multiple of the block size:
> copySize.width must be a multiple of blockWidth.
> copySize.height must be a multiple of blockHeight.

So if it's already a multiple, we shouldn't be uprounding it.

Reply:
Discussed on the matrix, we'll need to rehaul this for all backends as a follow-up